### PR TITLE
`[PRD]` Update compose governance copy + period input UI

### LINKED ISSUE #2541: Create Safe/Compose Governance copy updates
https://www.notion.so/hulking-cosmos-3e8/Q4-2024-QA-Requirements-13e2ee9c5bc280a88458e7a935d17488?pvs=4#13e2ee9c5bc28046b88ce7d721cc2ef9

### PR DESCRIPTION
**Pull request by DarksightKellar**
Closes #2541

Screenshots:
Screenshot 2024-11-26 at 11 40 42

Updated tooltip link style:
Screenshot 2024-11-26 at 12 10 10

### CODE REVIEW

File: src/components/DaoCreator/formComponents/AzoriusGovernance.tsx [hunk 1]
```diff
@@ -22,6 +27,64 @@ import { StepWrapper } from '../StepWrapper';
...
+          {direction === 'inc' ? <Plus size="1.5rem" /> : <Minus size="1.5rem" />}
+        </Button>
+      );
+
+      return (
+        <NumberInput
+          value={value}
+          onChange={onChange}
+          min={0}
+          focusInputOnChange={true}
```
Comment (mudrila) [inline src/components/DaoCreator/formComponents/AzoriusGovernance.tsx:55]: Can remove `={true}`

File: src/components/DaoCreator/formComponents/AzoriusGovernance.tsx [hunk 2]
```diff
@@ -22,6 +27,64 @@ import { StepWrapper } from '../StepWrapper';
...
 import { DAOCreateMode } from './EstablishEssentials';
 
+function DayStepperInput({
+  inputValue,
+  onInputChange,
+}: {
+  inputValue: number;
+  onInputChange: (val: number) => void;
+}) {
+  const MemoizedNumberStepperInput = memo(
```
Comment (mudrila) [inline src/components/DaoCreator/formComponents/AzoriusGovernance.tsx:37]: Why are we memoizing this and not just creating separate component?

**Review by mudrila — APPROVED** [review]
Couple of questions / minor things, but otherwise - looks good sir!

**Comment by parkermccurley** [comment]
I think we can remove the tooltips altogether tbh. Your call @DarksightKellar. Approved

File: src/components/DaoCreator/formComponents/AzoriusGovernance.tsx (hunk 2 above)
Comment (Da-Colon) [reply src/components/DaoCreator/formComponents/AzoriusGovernance.tsx:37]: same question

**Review by Da-Colon — APPROVED** [review]

**Comment by DarksightKellar** [comment]
> I think we can remove the tooltips altogether tbh. Your call @DarksightKellar.

Yep, agreed. Removed

**Pull request merged**